flocker/control/_model.py

```diff
@@ -90,9 +90,7 @@ class IRestartPolicy(Interface):
 
 
 @implementer(IRestartPolicy)
-@attributes([], apply_immutable=True,
-            # https://github.com/hynek/characteristic/pull/22
-            apply_with_init=False)
+@attributes([Attribute("name", default_value=u"never")], apply_immutable=True)
```

Comment:
I don't think this should be done this way. We want to separate model details from external API.

For the configuration (see `_config.py`) we had a mapping; probably want to something similar in `httpapi.py` (or share code with `_config.py`).